src/components/Invoice/newInvoice.js

```diff
@@ -0,0 +1,254 @@
+import React, { Component } from 'react';
+import { Button, Header, Icon, Modal, Form, Grid } from 'semantic-ui-react';
+import DatePicker from 'react-datepicker';
+import { CountryDropdown, RegionDropdown } from 'react-country-region-selector';
+import ReactTable from 'react-table';
+import moment from 'moment';
+
+import 'react-datepicker/dist/react-datepicker.css';
+import "react-table/react-table.css";
+import './newInvoice.css';
+
+
+class NewInvoice extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalOpen: false,
+      invoice: {
```

Comment:
In the future, this model will need to be more dynamic so that we can add fields easily. I don't know *how* we'll do this yet, so it's just something to keep in mind as you work in this area.